content/en/monitors/status/graphs.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Status Graphs
+further_reading:
+- link: "monitors/configuration/?tab=thresholdalert"
+  tag: "Documentation"
+  text: "Monitor configuration"
+- link: "monitors/configuration/?tab=thresholdalert/#group-retention-time"
+  tag: "Documentation"
+  text: "Group retention time"
+- link: "monitors/configuration/?tab=thresholdalert/#set-alert-conditions"
+  tag: "Documentation"
+  text: "Set alert conditions"
+---
+
+<div class="alert alert-info">Status Graphs is part of the <a href="/monitors/status/status_page">provisional Monitor Status Page</a>. If you are using the legacy status page, see the <a href="/monitors/status/status_legacy">Status Page (Legacy)</a> documentation.</div>
+
+## Overview
+
+The graphs on the monitor status page provide insights into the single monitor evaluations. These graphs help you to understand why your monitor might be in an ALERT state and where to focus your troubleshooting efforts.
+
+## Monitor metadata
+
+{{< img src="/monitors/status/graphs/graphs_metadata.png" alt="Monitor metadata section on the right side of the status page." style="width:100%;" >}}
+
+The right panel of the graph section of the monitor status page provides a high level overview of your monitor including
+
+|  | Description |
+| ---- | ---- |
+| Groups  | Count of groups per status (Alert, Warn, No Data, Ok)  |
+| Visualize as | Graph selector to toggle between the Evaluated Data, Source Data, and In Group graphs. |
```

Comment:
> and In Group graphs.

to replace with "and Transitions graphs"

Reply:
```suggestion
| Visualize as | Graph selector to toggle between the Evaluated Data, Source Data, and Transition graphs. |
```